src/components/AssetIcon.tsx

```diff
@@ -93,7 +97,7 @@ export const AssetIcon = memo(({ assetId, showNetworkIcon, src, ...rest }: Asset
   }
 
   if (assetId) {
-    if (asset?.icons) {
+    if (asset?.icons?.length) {
```

Comment:
Shouldn't be needed now that Portals gen has been updated not to generate an empty `icons` array but might as well for paranoia